multichain/plan.sh

```diff
@@ -0,0 +1,51 @@
+pkg_name=multichain
+pkg_origin=core
+pkg_version=1.0.8
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("GPL-3.0-only")
+pkg_description="MultiChain is an open source platform for private blockchains."
+pkg_upstream_url="https://www.multichain.com"
+pkg_source="https://www.multichain.com/download/${pkg_name}-${pkg_version}.tar.gz"
+pkg_shasum=5905bc7f236d0f7ff3978bbc6e791cafdfb34e8ca5b2f5fe5fca535a1336d575
+pkg_bin_dirs=(bin)
+pkg_deps=(
+  core/glibc
+  core/gcc-libs
+  core/patchelf
+)
+pkg_exports=(
+  [port]=system.port
+  [rpcport]=system.rpcport
+)
+pkg_exposes=(port rpcport)
+pkg_binds_optional=(
+  [multichain]="port rpcport"
+)
+# pkg_svc_user="hab"
```

Comment:
I think these comments can be removed :smile:

Reply:
Done!